### ISSUE TITLE
Do you still need this account?

### ISSUE THREAD

**Issue by gflk** [gf/two-letter-username#1]
Hi there, do you still need this account? My initials are gf and I would love to have them as my user name!

**Comment by taadis** [comment]
rt +1

**Comment by 9-k** [comment]
@taadis According to this post [here](https://awolski.com/request-your-github-username/) you can contact GitHub support and cite the [username policy](https://docs.github.com/en/free-pro-team@latest/github/site-policy/github-username-policy) to hopefully free up the username! This account hasn't been used for ages so I bet they'd deactivate it and let you take it. Either way, good luck!

**Comment by zvmu** [comment]
> Hi there, do you still need this account? My initials are gf and I would love to have them as my user name!

Hey do you need yours that is such a og name

**Comment by 9-k** [comment]
@7xh Yeah, I'm still using this one lol. But I did modify some code i found on github to generate some cool, still open, short usernames. Some of the ones remaining are l-q,
1-c
1-g
1-u
1-v
2-e
2-r
2-w
3-e
3-i
3-j
3-o
3-q
3-v
4-c
4-e
4-f
4-i
4-l
4-o
4-p
4-q
4-r
4-t
4-w
4-z
5-a
5-b.
Most of the other _-_ names are taken, lol. Good luck!

**Comment by 7xh** [comment]
> @7xh Yeah, I'm still using this one lol. But I did modify some code i found on github to generate some cool, still open, short usernames. Some of the ones remaining are l-q,
> 1-c
> 1-g
> 1-u
> 1-v
> 2-e
> 2-r
> 2-w
> 3-e
> 3-i
> 3-j
> 3-o
> 3-q
> 3-v
> 4-c
> 4-e
> 4-f
> 4-i
> 4-l
> 4-o
> 4-p
> 4-q
> 4-r
> 4-t
> 4-w
> 4-z
> 5-a
> 5-b.
> Most of the other _-_ names are taken, lol. Good luck!

lol not working i-q
l-q

**Comment by 9-k** [comment]
> > @7xh Yeah, I'm still using this one lol. But I did modify some code i found on github to generate some cool, still open, short usernames. Some of the ones remaining are l-q,
> > 1-c
> > 1-g
> > 1-u
> > 1-v
> > 2-e
> > 2-r
> > 2-w
> > 3-e
> > 3-i
> > 3-j
> > 3-o
> > 3-q
> > 3-v
> > 4-c
> > 4-e
> > 4-f
> > 4-i
> > 4-l
> > 4-o
> > 4-p
> > 4-q
> > 4-r
> > 4-t
> > 4-w
> > 4-z
> > 5-a
> > 5-b.
> > Most of the other _-_ names are taken, lol. Good luck!
> 
> lol not working i-q
> l-q

Looks like user "l-q" joined 22 days ago. I made this around new years, so it doesn't surprise me some of these are taken.

**Comment by 4-f** [comment]
> > > @7xh Yeah, I'm still using this one lol. But I did modify some code i found on github to generate some cool, still open, short usernames. Some of the ones remaining are l-q,
> > > 1-c
> > > 1-g
> > > 1-u
> > > 1-v
> > > 2-e
> > > 2-r
> > > 2-w
> > > 3-e
> > > 3-i
> > > 3-j
> > > 3-o
> > > 3-q
> > > 3-v
> > > 4-c
> > > 4-e
> > > 4-f
> > > 4-i
> > > 4-l
> > > 4-o
> > > 4-p
> > > 4-q
> > > 4-r
> > > 4-t
> > > 4-w
> > > 4-z
> > > 5-a
> > > 5-b.
> > > Most of the other _-_ names are taken, lol. Good luck!
> > 
> > 
> > lol not working i-q
> > l-q
> 
> Looks like user "l-q" joined 22 days ago. I made this around new years, so it doesn't surprise me some of these are taken.

i got one
